src/image/ImageAnnotator.ts

```diff
@@ -49,7 +49,8 @@ export default class ImageAnnotator extends BaseAnnotator {
 
         if (this.managers.size === 0) {
             this.managers.add(new PopupManager({ referenceEl }));
-            this.managers.add(new RegionManager({ referenceEl }));
+            this.managers.add(new RegionListManager({ referenceEl }));
```

Comment:
Should the order of these be flipped?

Reply:
No, because if we do that then when the mode is `region` the existing regions won't receive pointer events